packages/local_auth/local_auth/README.md

```diff
@@ -252,6 +252,37 @@ types (such as face scanning) and you want to support SDKs lower than Q,
 _do not_ call `getAvailableBiometrics`. Simply call `authenticate` with `biometricOnly: true`.
 This will return an error if there was no hardware available.
 
+#### Android theme
+
+On Android, ensure that the `LaunchTheme` of your app has a parent style with the `Theme.AppCompat.Light` theme; otherwise the app may crash. To do that go to `android > app > src > main > res > values > styles.xml` and look for the style with name `LaunchTheme` (Notice that `LaunchTheme` must be referenced in the `AndroidManifest.xml` file). Then, change the parent style with the new value and you should have set up something like this:
+
+```xml
+...
+<resources>
+  <style name="LaunchTheme" parent="Theme.AppCompat.Light">
+    ...
+  </style>
+  ...
+</resources>
+...
+```
+
+In case you don't have a `styles.xml` file for your Android project you can set up the Android theme directly on your `android > app > src > main > AndroidManifest.xml` file:
```

Comment:
Nit: remove "your" and "file" at the end; the fact that it's a file in their project is clear from context.